commands/download.bee.inc

```diff
@@ -19,7 +19,7 @@ function download_bee_command() {
       'multiple_argument' => 'projects',
       'options' => array(
         'hide-progress' => array(
-          'description' => bt('Hide the download progress bar.'),
+          'description' => bt('Deprecated, will get removed in a future version.'),
```

Comment:
We use this in tests MultisiteDownloadCommandsTest and DownloadCammandsTest
While Bee works fine; it just ignores options it doesn't recognise, we should tidy up while we're removing the option.

Reply:
Tests updated. :+1:

To make sure: the deprecation message in descriptions is OK? Or should the option get removed right away?

Reply:
Yes, I think the deprecation message makes sense. Bee will pass it to the command callback but it simply won't be used. This will help people know that it is no longer required in their script in the absence of anything like release notes.